### PR TITLE
Oger: init at 1.1.3 with dependencies

### PR DESCRIPTION
**Pull request by nico202**

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -3931,6 +3931,27 @@ in modules // {
     };
   };
 
+  MDP = buildPythonPackage rec {
+    version = "0.1.0";
+    name = "MDP-${version}";
+
+    src = pkgs.fetchurl {
+      url = https://pypi.python.org/packages/source/M/MDP/MDP-3.5.tar.gz;
```
Comment (joachifm) [inline pkgs/top-level/python-packages.nix]: It looks like the version here and in the `version` attribute disagree, is that intentional?

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -3931,6 +3931,27 @@ in modules // {
...
+      sha256 = "0aw1zxmyvx6gfmmnixbqmdaah28jl7rmqkzhxv53091asc23iw9k";
+    };
+
+    propagatedBuildInputs = with self; [ future numpy ];
+
+    doCheck = false;
```
Comment (joachifm) [inline pkgs/top-level/python-packages.nix]: Could you add a comment explaining why the tests have been disabled?

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (nico202) [inline pkgs/top-level/python-packages.nix]: Wasn't, sorry

File: pkgs/top-level/python-packages.nix (hunk 2 above)
Comment (nico202) [inline pkgs/top-level/python-packages.nix]: Enabled, was failing just because of missing pytest

**Pull request merged**